hack/make/test-buildkit

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+source hack/make/.integration-test-helpers
+
+if [ ! -z "${TEST_SKIP_BUILDKIT}" ]; then
+	echo buildkit tests skipped according to TEST_SKIP_BUILDKIT
+	exit 0
+fi
+
+(
+	env
+	bundle .integration-daemon-start
+
+	export BIN_PATH="${PWD}/bundles/dynbinary-daemon"
+	export PATH="${BIN_PATH}:${PWD}/bundles/binary-daemon:$PATH"
+
+	# Download buildkit
+	# TODO: change this to moby version of the project after
+	# https://github.com/moby/buildkit/pull/1164 is merged.
+	echo "curl -fsSL -o ${PWD}/buildkit.tar.gz https://github.com/SamWhited/buildkit/archive/${BUILDKIT_COMMIT}.tar.gz"
```

Comment:
Instead of using `echo` here, you can use this to show the commands that are being run if the `TESTDEBUG` env-var is set;

```bash
[ -n "${TESTDEBUG}" ] && set -x
```

Reply:
Done